dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/datavalueset/DataValueSet.java

```diff
@@ -50,7 +50,7 @@
 @Setter
 @NoArgsConstructor
 @JacksonXmlRootElement( localName = "dataValueSet", namespace = DxfNamespaces.DXF_2_0 )
-public final class DataValueSet
+public class DataValueSet
```

Comment:
Is there a particular reason to remove the `final`? I did not see a new subclass and having worked hard to get rid of them I would like to keep it that way :D

Reply:
Right sorry for that, it was for mocking and testing purposes. Mockito does not allow mocking of final classes. There could be a better way to do this, any ideas?

Reply:
No, that is fine. Just wanted to know if there are actual sub-classes.

Reply:
Understand.